src/benchmarks/micro/libraries/System.Formats.Tar/Perf.Tar.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Formats.Tar.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    public class Perf_TarWriter
+    {
+        private static readonly string _fileName = "file.txt";
+        private static Dictionary<string, string> _ea;
+        private static MemoryStream _memoryStream;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _memoryStream = new MemoryStream();
+        }
+
+        [GlobalSetup(Targets = new[] { nameof(PaxTarEntry_WriteEntry), nameof(PaxTarEntry_WriteEntry_Async) })]
+        public void SetupPax()
+        {
+            _ea = new Dictionary<string, string>
```

Comment:
We can have only one setup per benchmark, so `SetupPax` needs to call `Setup` in explicit way

```suggestion
            Setup();

            _ea = new Dictionary<string, string>
```